src/util/substrateProviders.ts

```diff
@@ -52,8 +65,32 @@ export const customTypes: CustomTypes = {
   [CHAIN_2]: types[CHAIN_2]
 };
 
+export const bridgeIds: GenericKeyStringConfigs = {
+  [CHAIN_1]: 'rlto',
+  [CHAIN_2]: 'mlau'
+};
+
 export const customHashers: CustomHashers = {
   [CHAIN_2]: hasherH512
 };
 
+export const ss58Formats: GenericKeyNumberConfigs = {
+  [CHAIN_1]: 48,
+  [CHAIN_2]: 60
+};
+
+export const accountDerivations: GenericKeyStringConfigs = {
+  [CHAIN_1]: 'pallet-bridge/account-derivation/account',
+  [CHAIN_2]: 'pallet-bridge/account-derivation/account'
+};
```

Comment:
This is going to be always constant, so no need to have per-chain configuration.

Reply:
cool ...